dataactbroker/handlers/accountHandler.py

```diff
@@ -660,52 +660,54 @@ def listUsersWithStatus(self):
             userInfo.append(thisInfo)
         return JsonResponse.create(StatusCode.OK,{"users":userInfo})
 
-    def listSubmissionsByCurrentUserAgency(self):
-        """ List all submission IDs associated with the current user's agency """
+    def list_submissions(self, page, limit, certified):
+        """ List submission based on current page and amount to display. If provided, filter based on
+        certification status """
+
+        # convert params and type check
+        try:
+            page = int(page) if page is not None else 1
+        except:
+            raise ValueError("Incorrect type specified for 'page'. Please enter a positive number.")
+
+        try:
+            limit = int(limit) if limit is not None else 5
+        except:
+            raise ValueError("Incorrect type specified for 'limit'. Please enter a positive number.")
+
+        if certified is not None:
+            certified = certified.lower()
+        # If certified is none, get all submissions without filtering
+        if certified is not None and certified not in ['true', 'false']:
+            raise ValueError("Incorrect value specified for the 'certified' parameter")
```

Comment:
We talked about this a bit before, but looking at this now, do you think it'd make sense to move this type checking into the route function? It'd keep the HTTP handling/munging in the route and leave this function more pure.

Also, should you convert `certified` to a boolean rather than leaving it as a string?